schema/config.json

```diff
@@ -425,7 +427,7 @@
         },
         "customIcons": {
           "$ref": "#/$defs/CustomIconsConfig",
-          "description": "Custom icons for filenames and file extensions"
+          "description": "Custom icons for filenames and file extensions\nSee https://github.com/jesseduffield/lazygit/blob/master/docs/Config.md#custom-files-icon--color"
```

Comment:
@karimkhaleel I find it interesting that we have this description both on the `CustomIconsConfig` definition above, and here on the `$ref`. I suppose this one here isn't needed? Same problem with the `CommitLengthConfig` struct, and probably others. Must have come in with 30e9bf8a755d09.

It doesn't seem to be a problem in practice, but if you feel bored, you might look into whether this is our fault or invopop's. Shouldn't block this PR, of course.